experiment/jenkins_to_prow.py

```diff
@@ -27,7 +27,7 @@
 
 TEMPLATE = {
     'name': '',
-    'interval': '6h',
```

Comment:
looks like 1.5 jobs were run with this at 6h, and 1.6 jobs at 2h, was that the intent?